fcd_sale_order/migrations/14.0.1.1.0/pre-migration.py

```diff
@@ -0,0 +1,30 @@
+# © 2022 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+
+def migrate(cr, version):
```

Comment:
Better try to use `openupgrade` tools. See an example in https://github.com/solvosci/slv-manufacture/blob/13.0/stock_valuation_mrp/migrations/13.0.2.2.0/pre-migration.py , using `logged_query` will provide us more information even without low log levels configured.